From f6853fae52701dede431b07a51be4b39d152f5d7 Mon Sep 17 00:00:00 2001
From: Xu Bing <bing.xu@intel.com>
Date: Mon, 28 Apr 2025 12:16:48 +0800
Subject: [PATCH] Fix the issue of MapsPlaceholderActivity disappearing
 abnormally

MapsPlaceholderActivity will restart and it will be covered by
red suface, so change the color to green.

Tracked-ON: OAM-130875
Signed-off-by: Xu Bing <bing.xu@intel.com>
---
 app/src/com/android/car/carlauncher/CarLauncherViewModel.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/src/com/android/car/carlauncher/CarLauncherViewModel.java b/app/src/com/android/car/carlauncher/CarLauncherViewModel.java
index 3b2b0813..8834c10c 100644
--- a/app/src/com/android/car/carlauncher/CarLauncherViewModel.java
+++ b/app/src/com/android/car/carlauncher/CarLauncherViewModel.java
@@ -194,7 +194,7 @@ public final class CarLauncherViewModel extends ViewModel implements DefaultLife
                 // that nothing is wrong with the task view but maps
                 // in the task view has crashed. More details in
                 // b/247156851.
-                mRemoteCarTaskView.getValue().setBackgroundColor(Color.RED);
+                mRemoteCarTaskView.getValue().setBackgroundColor(Color.rgb(65, 175, 106));
             }
         }
     }
-- 
2.34.1

